Split cxx::runner test into separate tests

Having single monolith test is an extremely bad idea. No run separately, no debug etc

And maybe we need to replace cxxtest with some other testing framework. Which at minimum will provide more user friendly assertion messages, instead of:

```
3: In DServerMiscTestSuite::test_info_call:
3: /home/goetz/git/cppTango/cpp_test_suite/cxxtest/include/../../new_tests/cxx_dserver_misc.cpp:210: Error: Assertion failed: dserver->info().server_host == server_host
3: TEST FAILED in:
```

This should show actual values

--- a/test/cpp_test_suite/cxxtest/CMakeLists.txt
+++ b/test/cpp_test_suite/cxxtest/CMakeLists.txt
@@ -1,10 +1,14 @@
 macro(CXX_GENERATE_TEST name)
-    message("Generate ${name})")
+    message("Generate ${name}.cpp")
     execute_process(COMMAND python cxxtestgen.py --template=${CMAKE_CURRENT_SOURCE_DIR}/template/tango_template.tpl
             -o ${CMAKE_CURRENT_SOURCE_DIR}/${name}.cpp
             ../../new_tests/${name}.cpp
             WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/bin
             RESULT_VARIABLE FAILED)
+
+    if(${FAILED})
+        message(SEND_ERROR " Failed to generate ${name}.cpp. rv=${FAILED}")
+    endif()
 
     add_executable(${name} $<TARGET_OBJECTS:compare_test_object> ${name}.cpp)
     target_include_directories(${name} PRIVATE include)

